### PR TITLE
Add social buttons

### LINKED ISSUE #901: "I Just Bought Something on OpenBazaar" Social Media Sharing Tool
As a user, when I purchase something, I want an easy way to immediately send a message to social media about my purchase.

It should explain that I just made a purchase on OpenBazaar and let me select between 2 types of messages, a "general" message, such as "I just made a purchase on OpenBazaar!" with a link to download OpenBazaar OR a "specifc" message that shares the item I bought, that I bought it on OpenBazaar, and have a link to the item.

### PR DESCRIPTION
**Pull request by jjeffryes**
Adds social sharing buttons to the purchase complete screen.

Closes #901

### CODE REVIEW

File: js/templates/modals/purchase/complete.html [hunk 1]
```diff
@@ -18,7 +18,33 @@
     </div>
     <div class="col2"></div>
   </div>
-  <hr class="clrBr rowHg">
+  <div class="contentBox clrP clrBr padMd">
+    <h5><%= ob.polyT('purchase.completeSection.share.title') %></h5>
+    <p><%= ob.polyT('purchase.completeSection.share.body', { link: ob.polyT('purchase.completeSection.share.link') }) %></p>
+    <div class="flex flexKidsExpand gutterH">
+      <% var shareURL = "https%3A%2F%2Fopenbazaar.org" %>
```
Comment (rmisio) [inline js/templates/modals/purchase/complete.html]: Is it necessary for the backslashes escaped here?

**Comment by morebrownies** [comment]
@jjeffryes @rmisio  do either of you happen to have a testnet store running that I can test an order on?

File: js/templates/modals/purchase/complete.html [hunk 2]
```diff
@@ -18,7 +18,33 @@
...
+        </span>
+      </a>
+      <a class="btn btnThin clrP clrBr" href="https://www.facebook.com/sharer/sharer.php?u=<%= shareURL %>">
+        <span class="flexInline gutterHSm">
+          <i class="ion-social-facebook facebookColor"></i><span><%= ob.polyT('purchase.completeSection.share.postToFacebook') %></span>
+        </span>
+      </a>
+      <a class="btn btnThin clrP clrBr" href="https://pinterest.com/pin/create/button/?url=<%= shareURL %>&description=<%= ob.polyT('purchase.completeSection.share.shareMsg') %>">
+        <span class="flexInline gutterHSm">
+          <i class="ion-social-pinterest pinterestColor"></i><span>P<%= ob.polyT('purchase.completeSection.share.postToPinterest') %></span>
```
Comment (rmisio) [inline js/templates/modals/purchase/complete.html]: You have an extra `P` there which renders the button as `PPost to Pinterest`. (makes for some interesting alliteration)

File: js/templates/modals/purchase/complete.html [hunk 3]
```diff
@@ -18,7 +18,33 @@
     </div>
     <div class="col2"></div>
   </div>
-  <hr class="clrBr rowHg">
+  <div class="contentBox clrP clrBr padMd">
+    <h5><%= ob.polyT('purchase.completeSection.share.title') %></h5>
+    <p><%= ob.polyT('purchase.completeSection.share.body', { link: "<a class='clrTEm' href='https://openbazaar.org'>https://openbazaar.org</a>" }) %></p>
+    <div class="flex flexKidsExpand gutterH">
+      <% var shareURL = "https%3A%2F%2Fopenbazaar.org" %>
```
Comment (rmisio) [inline js/templates/modals/purchase/complete.html]: Is it necessary to escape those slashes?

**Comment by rmisio** [comment]
@morebrownies As a matter of fact:
ob://QmULuYvD7Am9gF3AYbbJMsES8MjBHDFvyt5h9aMRVR6eqN/store/matter-of-fact

**Comment by rmisio** [comment]
@morebrownies @jjeffryes Should the `Learn more` sentence have a period at the end (with the period not being a part of the link)?

image

File: js/templates/modals/purchase/complete.html (hunk 3 above)
Comment (jjeffryes) [reply js/templates/modals/purchase/complete.html]: According to the docs it is.

**Comment by rmisio** [comment]
@jjeffryes This is probably minor and we could let it go or circle back to it, but if the translations of the button text gets long, is that layout ok?

image

Probably really extreme case:

image

File: js/templates/modals/purchase/complete.html (hunk 2 above)
Comment (jjeffryes) [reply js/templates/modals/purchase/complete.html]: Good catch.

**Comment by morebrownies** [comment]
> @morebrownies @jjeffryes Should the Learn more sentence have a period at the end (with the period not being a part of the link)?

Yeah.

> @jjeffryes This is probably minor and we could let it go or circle back to it, but if the translations of the button text gets long, is that layout ok?

Some vertical padding would help a bit, but otherwise think it is fine.

**Comment by morebrownies** [comment]
It would also be worth adding a `#bitcoin` hashtag to the Tweet text @jjeffryes . #bitcoin is searched a lot of Twitter, and OpenBazaar would benefit from frequently displaying in those searches.

screenshot 2017-09-26 15 59 27

screenshot 2017-09-26 15 59 36

**Comment by morebrownies** [comment]
Other than that, looks good to me.

**Comment by morebrownies** [comment]
Very excited to get this feature integrated 👍

**Comment by jjeffryes** [comment]
Ok, this is updated.

**Comment by morebrownies** [comment]
👍

**Comment by morebrownies** [comment]
Works well, just tried it here:

https://twitter.com/thee_wolf/status/912821199540912128

**Pull request merged**